nautobot_netbox_importer/management/commands/import_netbox_objectchange_json.py

```diff
@@ -0,0 +1,105 @@
+"""Definition of "manage.py import_netbox_objectchange_json" Django command for use with Nautobot."""
+import argparse
+import json
+
+from packaging import version
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand, CommandError
+from django.contrib.contenttypes.models import ContentType
+
+from nautobot.extras.models import ObjectChange
+
+from nautobot_netbox_importer.utils import ProgressBar
+from nautobot_netbox_importer.command_utils import validate_netbox_version, initialize_logger
+from nautobot_netbox_importer.diffsync.models.validation import netbox_pk_to_nautobot_pk
+
+
+User = get_user_model()
+
+
+class Command(BaseCommand):
+    """Implementation of import_netbox_objectchange_json command."""
+
+    help = "Import ObjectChange objects from NetBox YAML data dump into Nautobot's database"
+
+    def add_arguments(self, parser):
+        """Add parser arguments to the import_netbox_objectchange_json management command."""
+        parser.add_argument("json_file", type=argparse.FileType("r"))
+        parser.add_argument("objectchange_json_file", type=argparse.FileType("r"))
+        parser.add_argument("netbox_version", type=version.parse)
+        parser.add_argument("--dryrun", type=bool, default=False)
+
+    def handle(self, *args, **options):
+        """Handle execution of the import_netbox_json management command."""
+
+        def process_objectchange(entry):
+            """Processes on objectchange entry to migrate from Netbox to Nautobot."""
+            try:
+                entry["fields"]["changed_object_type"] = ContentType.objects.filter(
+                    id=nautobot_contenttype_mapping[netbox_contenttype_mapping[entry["fields"]["changed_object_type"]]]
+                ).first()
+            except KeyError:
+                logger.warning(
+                    f'{netbox_contenttype_mapping[entry["fields"]["changed_object_type"]]} key is not mapped'
+                )
+                return
+
+            if entry["fields"]["related_object_type"]:
```

Comment:
Do we need a `try...except KeyError` block here as well?

Reply:
I found that all the objects contain this keyword, but I will add to be sure not breaking with some old API serialization

Reply:
I was thinking for the case where the related_object_type was a ContentType not present in Nautobot?

Reply:
yup, gotcha